src/Command/MigrateCommand.php

```diff
@@ -505,13 +503,13 @@ private function migrate(
             } else if ($referenceType === self::AUTO) {
                 $referenceType = $this->guessReferenceType($assetCsvFilename);
             }
-            $this->io->writeln(sprintf('The command will be ran with <options=bold>--reference-type=%s</>', $referenceType));
+            $this->io->writeln(sprintf('The command will be run with <options=bold>--reference_type=%s</>', $referenceType));
```

Comment:
```suggestion
            $this->io->writeln(sprintf('The command will run with <options=bold>--reference_type=%s</>', $referenceType));
```

Reply:
Camille has already reviewed the english

Reply:
ouki